tests/conftest.py

```diff
@@ -1,8 +1,14 @@
+import platform
 from typing import Dict, List
```

Comment:
Please do like you did in the test file, with importing `annotations` from `__future__`.

Reply:
Yeah I think it was necessary to include `from __future__ import annotations` because the type hinting style we are using was introduced in python 3.8 or so, and the project depends on python 3.6 or greater, so in order not to get errors in the later versions, importing `annotations` from `__future__` is a good idea since it converts all type annotations to `str`.